api/src/main/java/org/pmiops/workbench/cdr/CdrVersionService.java

```diff
@@ -61,13 +56,22 @@ public CdrVersionService(
    * @param version
    */
   public void setCdrVersion(DbCdrVersion version) {
-    // TODO: map data access level to authorization domain here (RW-943)
-    String authorizationDomain = configProvider.get().firecloud.registeredDomainName;
+    if (!accessTierService
```

Comment:
A much faster check than calling Terra/Firecloud to verify the Auth Domain - and equivalent.  We may want to remove the Auth Domain check at some point in the future.

Reply:
Thanks for not changing that in this PR - I think we'd need a discussion about source of truth before we did that.